DEPRECATED.md

```diff
@@ -199,14 +199,7 @@ You can use the static helper methods such as `System::loadLanguageFile()` or
 
 ## Constants
 
-The constants `TL_ROOT` and `TL_MODE` have been deprecated and will be removed
-in Contao 5.0.
-
-Use the `kernel.project_dir` instead of `TL_ROOT`:
-
-```php
-$rootDir = System::getContainer()->getParameter('kernel.project_dir');
-```
+The constant `TL_MODE` has been deprecated and will be removed in Contao 5.0.
```

Comment:
This seems wrong :)

Reply:
Why? this PR is only about `TL_ROOT` that’s why I only removed `TL_ROOT` from here and not `TL_MODE` because thats part of #4741

Reply:
Ah I see. 🙈